.husky/pre-commit

```diff
@@ -1,7 +1 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
-LATEST_HASH=$(git rev-parse --verify HEAD)
-
-# Run commands only in packages with changed files compared to previous commit and on their dependants in parallel
-pnpm check:all --filter="...[$LATEST_HASH]"
+pnpm
```

Comment:
This was actually automatically changed via husky upgrade, hadn't noticed. Honestly though, I'm in favour of removing these checks, I use `--no-verify` every time because I can't be bothered to wait.